pilot-app/src/components/form/fields/wimTagVue.vue

```diff
@@ -0,0 +1,89 @@
+﻿<template>
+    <div :class="classname">
+        <label v-if="prefix(field)" v-html="prefix(field)"></label>
+        <vue-tags-input v-model="tag"
+                        :tags="tags"
+                        :autocomplete-min-length="0"
+                        :autocomplete-items="filteredItems"
+                        @tags-changed="tagsChanged"
+                        v-bind:disabled="field.disabled || field.readOnly" />
+        <label v-if="suffix(field)" v-html="suffix(field)"></label>
+    </div>
+</template>
+<script>
+    import { fieldMixin } from './../form';
+    import VueTagsInput from '@johmun/vue-tags-input';
+
+
+    export default {
+        props: ['field', 'classname'],
+        mixins: [fieldMixin],
+        components: {
+            VueTagsInput
+        },
+        computed: {
+            filteredItems() {
+                return this.autocompleteItems.filter(i => {
+                    return i.text.toLowerCase().indexOf(this.tag.toLowerCase()) !== -1;
+                });
+            },
+            autocompleteItems() {
+                return this.field.options.filter((o) => o.text).map((option) => {
+                    return {
+                        text: option.text
+                    };
+                });
+            },
+            tags: {
+                get() {
+                    if (this.field.value) {
+                        return this.field.value.map((v) => {
+                            let option = this.field.options.find((o) => o.value === v);
+                            let text = ""
+                            if (v) {
+                                if (typeof (v) === "string") {
+                                    text = v;
+                                }
+                                else if (typeof (v) === "object") {
+                                    v.text
+                                }
+                            }
+
+                            return {
+                                text: option ? option.text : text,
+                                tiClasses: ["ti-valid"]
+                            };
+                        });
+                    }
+                    return [];
+                },
+                set(value) {
+                    // console.log(value);
+                    this.field.value = value.map(r => r.text);
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
                    this.field.value = value.map(r)r => r.text);
```